charter.md

```diff
@@ -0,0 +1,22 @@
+# Open Web Docs Charter
+
+## Mission
+
+Open Web Docs supports web platform documentation for the benefit of web developers & designers worldwide.
+
+
+## Core Values
+
+Open Web Docs believes open web platform documentation is critical digital infrastructure that benefits everyone. To demonstrate that belief, we: 
+
+- _focus_ on documentation that has a positive impact on end-users and web developers.
+- _prioritize_ based on needs of the global community of web developers and designers. 
+- _maintain_ an independent editorial voice, with a “firewall” between fiscal and editorial matters. 
+- _adhere_ to open source principles of transparency in project governance, investment, and feature development in order to maintain trustworthiness and credibility with the web community. 
+- _work_ to make the platform more equitable, investing in documenting internationalization, accessibility, privacy and diversity components of the web. 
```

Comment:
"...privacy and diversity components of the web" -> "...privacy, diversity, and other critical components of a healthy World Wide Web"...?

Reply:
Also maybe not mention diversity here, as you cover it in the below bullet too?